History feature

Kanchi11 opened the pull request:
Made changes to the ui of history page

BrandonTroy reviewed (changes requested):
Looks pretty good! I love that you included a tag for what the most common sentiment was.

Would it be possible to also include a tag for the "type" of each listing. This would just be the same as the name as whatever section it would have appeared in in the old UI (ie. "Product", "Image", "Live Speech").

Also, the hover effect on the entire row is misleading, because that implies that clicking it would do something, but it doesn't. Also the orange becomes a little overkill with that effect. I would get rid of that and instead add a hover effect on the view button, which does do something.

ChristopherElchik reviewed (changes requested):
Looks solid. Also-- instead of merging straight into master, can you switch it to merge to development? That way we can handle all merging before doing development --> master

Kanchi11 commented:
I have changed the history item hover and applied it to view button. Also changed the hover on nav-links on the nav bar.  The responsiveness of the index page is also fixed. Check and let me know !

BrandonTroy reviewed (approved):
Just added some small changes such as making the analysis type attribute display as a badge, and then combining everything into one table because we now display the type within the entry, and fixing the image scaling in the banner.

ChristopherElchik reviewed (approved):
Looks solid
